html/js/folksonomy.js

```diff
@@ -146,7 +146,7 @@ function folskonomy_engine_init() {
  */
 function displayFolksonomyPropertyValues() {
     //$(".details").before(
-    $("div[itemtype='https://schema.org/Product']").append(
+    $("div.main-product").append(
```

Comment:
Not sure if it fits this PR, but if used in this way, it might be better to make it `div id="main-product"` instead of a `class`?

Reply:
you are right, I will do both to avoid disturbing CSS :-)